Cargo.toml

```diff
@@ -14,7 +14,7 @@ name = "elf2tab"
 path = "src/main.rs"
 
 [dependencies]
-chrono = "0.4.2"
+chrono = { version = "0.4.2", default-features = false, features = ["clock", "std"] }
```

Comment:
Why not just update chrono to the latest version instead?

Reply:
The [latest `chrono`](https://crates.io/crates/chrono/0.4.23/dependencies) still has the optional `time` dependency that cargo audit complains about.

Reply:
Interesting, it seems like Chrono devs think it [isn't vunrable](https://github.com/chronotope/chrono/issues/602) and will remove the `oldtime` feature in the next version as well